### PR TITLE
Put Election API Call

### LINKED ISSUE #124: ✨ On Election Wizard Completion Submit to API
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Suggestion

When the election setup wizard completes submit all data to the create election API endpoint

### Possible Implementation

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lprichar**
### Issue

Resolves #124 

### Description
When you complete the election setup wizard send the data into the API in order to create an election.

### Testing
1. Log in
2. Go through wizard
3. Once you get to the confirmation page click submit and you should get a loading screen for a sec:

image

Then it should show a success page:

image

4. Hit Return to Election List

Then it should redirect to the election list and your new election should be listed.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (keithrfung) [inline .vscode/launch.json:2]: JSON doesn't take comments. Should probably pull these or add to Readme.

**Review by keithrfung — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (lprichar) [reply .vscode/launch.json:2]: While true, that file was generated by and used exclusively by VSCode, so I think it's ok, but I'll remove

File: packages/admin-app/src/components/ElectionSetupWizard/Steps/JointKeySelectStep.tsx [hunk 2]
```diff
@@ -59,25 +60,29 @@ const JointKeySelectStep: React.FC<JointKeySelectStepProps> = ({ onNext, onChang
...
+            });
+        };
 
-    const getKeyCeremonies = async () => {
-        await ceremonyClient.find(0, 100, { filter: {} }).then((response) => {
-            setKeyCeremonies(response.key_ceremonies);
-        });
-    };
+        findKeyCeremonies();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (lprichar) [inline packages/admin-app/src/components/ElectionSetupWizard/Steps/JointKeySelectStep.tsx:72]: @keithrfung as you can see I converted the client retrieval code into a hook, but the problem is hooks have to be at the root level and can't be called from inside other hooks.  So you have to pass it in to the array in the 2nd parameter of useEffect, but that produces infinite recursion.  The only solution I could find was this lint disable.  Got any ideas?

Comment (keithrfung) [reply packages/admin-app/src/components/ElectionSetupWizard/Steps/JointKeySelectStep.tsx:72]: This is probably the documentation you're looking for explaining the linting
https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Comment (keithrfung) [reply packages/admin-app/src/components/ElectionSetupWizard/Steps/JointKeySelectStep.tsx:72]: The ceremonyClient should not be changing. So by passing it in, it shouldn't be rerendering. If that is happening, there is something happening with the client itself. 

https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects

**Review by SteveMaier-IRT — APPROVED** [review]

**Pull request merged**